redisvl/redis/connection.py

```diff
@@ -143,11 +144,18 @@ def validate_modules(
             if int(installed_version) >= int(required_module["ver"]):  # type: ignore
                 return
 
-    raise ValueError(
-        f"Required Redis database module {required_module['name']} with version >= {required_module['ver']} not installed. "
-        "See Redis Stack documentation: https://redis.io/docs/stack/"
+    error_message = "Required Redis db module "
+
+    error_message += " OR ".join(
+        [f'{module["name"]} >= {module["ver"]}' for module in required_modules]
+    )
+
+    error_message += (
+        " not installed. See Redis Stack docs at https://redis.io/docs/stack/."
```

Comment:
this page now redirects to https://redis.io/docs/latest/operate/oss_and_stack/
 is this still the page we want linked here?

Reply:
Good catch. Updating!